newrelic.js

```diff
@@ -44,6 +44,18 @@ exports.config = {
    * attributes include/exclude lists.
    */
   allow_all_headers: true,
+  application_logging: {
+    /**
+     * When true, logs generated by your application will be forwarded to New Relic One.
+     */
+    enabled: true,
+    local_decorating: {
+      /**
+       * TODO
```

Comment:
The spec references it here https://source.datanerd.us/agents/agent-specs/pull/570/files#diff-22ea975b3218ea7ba317770796151db7a25e9e8a59ef0799d8223ee8506802c9R122

Reply:
👍